coda/coda_validate/tests/test_views.py

```diff
@@ -0,0 +1,376 @@
+import json
+
+from lxml import objectify
+import pytest
+
+from django.core.urlresolvers import reverse
+from django import http
+
+from .. import factories
+from .. import views
+from ..models import Validate
+
+VALIDATE_XML = '{http://digital2.library.unt.edu/coda/validatexml/}validate'
+
+pytestmark = [
+    pytest.mark.urls('coda_validate.urls'),
+    pytest.mark.django_db
+]
+
+
+class TestIndex:
+    """
+    Tests for coda_validate.views.index.
+    """
+
+    def test_status_ok(self, rf):
+        factories.ValidateFactory.create_batch(30)
+        request = rf.get('/')
+        response = views.index(request)
+        assert response.status_code == 200
+
+    def test_status_ok_without_validate_objects(self, rf):
+        request = rf.get('/')
+        response = views.index(request)
+        assert response.status_code == 200
+
+    def test_template_used(self, client):
+        response = client.get(reverse('coda_validate.views.index'))
+        assert response.template[0].name == 'coda_validate/index.html'
+
+    def test_context(self, client):
+        factories.ValidateFactory.create_batch(30)
+        response = client.get(reverse('coda_validate.views.index'))
+        context = response.context[-1]
+        assert len(context['recently_prioritized']) == 20
```

Comment:
I am not sure I follow. `recently_prioritized` is just the first 20 results of a query to get all Validate objects with a priority > 0 in reverse order by the change date.